### PR TITLE
Improved the auth/callback doc to solve the url bug

### PR DESCRIPTION
**Pull request by vinayakhegde1**
## What kind of change does this PR introduce?

This PR improves the doc to fix auth/callback url error #18730

## What is the current behavior?

 Issue #18730

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context
Updated the code as per the given improvement in #18730

### CODE REVIEW

File: apps/docs/pages/guides/auth/server-side/oauth-with-pkce-flow-for-ssr.mdx [hunk 1]
```diff
@@ -29,43 +29,43 @@ Create a new file at `app/auth/callback/route.ts` and populate with the followin
 
 ```ts app/auth/callback/route.ts
 import { createServerClient, type CookieOptions } from '@supabase/ssr'
```
Comment (Crayon-ShinChan) [inline apps/docs/pages/guides/auth/server-side/oauth-with-pkce-flow-for-ssr.mdx:31]: the import is duplicated

**Review by mengxi-ream — CHANGES_REQUESTED** [review]

File: apps/docs/pages/guides/auth/server-side/oauth-with-pkce-flow-for-ssr.mdx (hunk 1 above)
Comment (vinayakhegde1) [reply apps/docs/pages/guides/auth/server-side/oauth-with-pkce-flow-for-ssr.mdx]: @Crayon-ShinChan removed the line

**Review by mengxi-ream — APPROVED** [review]

**Review by charislam — APPROVED** [review]

**Comment by charislam** [comment]
Thanks @vinayakhegde1! Much appreciated.

Looks good, but it seems like your formatter might be different from our formatter as the Prettier check is failing. Do you mind reformatting this so we can merge?

**Comment by vinayakhegde1** [comment]
@charislam sure but if you can help me exactly how can I  achieve that

**Comment by charislam** [comment]
The check that is failing is this: `npx prettier -c '{apps,studio}/**/*.{js,jsx,ts,tsx,css,md,mdx,json}'`

If you run `npx prettier '{apps,studio}/**/*.{js,jsx,ts,tsx,css,md,mdx,json}'`, it should reformat the code to match our style guide. If that breaks things, you can also [prettier-ignore](https://prettier.io/docs/en/ignore.html) this section

**Comment by Oms-343** [comment]
hello @charislam  ,   @Crayon-ShinChan  I  have created the pull request for the same and added @vinayakhegde1 as a co-author here #18923 .

**Review by silentworks — APPROVED** [review]

**Comment by Crayon-ShinChan** [comment]
Actually doc here also need to be improved with the same problem https://supabase.com/docs/guides/auth/server-side/email-based-auth-with-pkce-flow-for-ssr

**Pull request merged**